sdks/python/apache_beam/dataframe/io.py

```diff
@@ -53,6 +53,7 @@
 from apache_beam import io
 from apache_beam.dataframe import frame_base
 from apache_beam.io import fileio
+from apache_beam.io.fileio import WriteToFiles
```

Comment:
no need to import since you can follow the existing way like fileio.WriteToFiles.